src/components/overview/ImageGallery.jsx

```diff
@@ -1,11 +1,58 @@
-import React from 'react';
+import React, {useState} from 'react';
 import styles from './overview.module.css'
+import json from './styleSample.json';
 
 
 const ImageGallery = (props) => {
 
+  var nums = Array.from(Array(7).keys()) //Used for testing flex box children size
+  // console.log(nums)
+
+  /*Gallery would need
+   -state of style index,
+    -style information,
+    chosen thumbnail index
+    */
+
+
+   var photos = json.results[0].photos;
+  // const [selectedStyle, setStyle] = useState() //Place holder for use of context to change style
```

Comment:
This line will no longer be needed when you start using ProductContext?

Reply:
Place holder for next step of implementation. But no longer needed. Removed comment.

Reply:
Removed comment.